exe/puppet-db.cc

```diff
@@ -40,6 +41,16 @@ main(int argc, char **argv) {
         po::options_description global_options("global options");
         global_options.add_options()
                 ("help,h", "produce help message")
+                ("config", po::value<string>()->default_value(""),
```

Comment:
One thing to note here, long GNU options are not loved by everyone in the unix world, in fact there is a set of admins prefer the short options for succinctness. This is why most programs have support for long and short options, to appease everyone. Having to type puppet foo --config 'bar.cfg' versus puppet foo -c 'bar.cfg' for example.